deploy/dependencies/drone-ci/bin/deploy-drone.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+cd "$( dirname "${BASH_SOURCE[0]}" )"/../
+
+read -p "clientID: " clientID
+read -p "clientSecret: " clientSecret
+read -p "admin github: " admin
+
+helm upgrade --install drone-secret-release ./drone --namespace drone-ci \
+  -f drone/values/values.yml \
+  --set sourceControl.github.clientID=$clientID \
+  --set clientSecret=$clientSecret \
+  --set server.adminUser=$admin
+helm upgrade --install drone-release stable/drone --namespace drone-ci \
```

Comment:
why do we need secret in k8s and we do we pass same data to secret and to drone release?